MantidPlot/src/ProjectRecoveryGUIs/RecoveryFailureView.h

```diff
@@ -8,23 +8,20 @@
 #define RECOVERYFAILUREVIEW_H
 
 #include "ProjectRecoveryPresenter.h"
+#include "ui_RecoveryFailure.h"
```

Comment:
Please check if the `.cpp` file is still including this file too.

Reply:
Same as above